setup.cfg

```diff
@@ -0,0 +1,61 @@
+[metadata]
+name = pycti
+version = attr: pycti.__version__
+author = OpenCTI
+author_email = contact@opencti.io
+maintainer = OpenCTI
+url = https://github.com/OpenCTI-Platform/client-python
+description = Python API client for OpenCTI.
+long_description = file: README.md
+long_description_content_type = text/markdown
+license = Apache
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Intended Audience :: Developers
+    Intended Audience :: Information Technology
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: Apache Software License
+    Natural Language :: English
+    Natural Language :: French
+    Operating System :: OS Independent
+    Programming Language :: Python :: 3
+    Topic :: Security
+    Topic :: Software Development :: Libraries :: Python Modules
+
+[options]
+python_requires = >=3.7
+packages =
+    pycti
+    pycti.api
+    pycti.connector
+    pycti.entities
+    pycti.utils
+include_package_data = True
+# https://github.com/pypa/setuptools/issues/1951
+install_requires =
+    datefinder~=0.7
+    pika~=1.2
+    python-magic-bin~=0.4.14; sys_platform == "win32"
+    python-magic~=0.4.24; sys_platform == "linux" or sys_platform == "darwin"
+    python_json_logger~=2.0.2
+    pyyaml~=6.0
+    requests>=2.25
+    setuptools~=60.1.0
+    sseclient~=0.0.27
+    stix2~=3.0.1
+
+[options.extras_require]
+dev =
+    black==22.1.0
+    pre-commit~=2.13
+    pytest~=6.2
+    pytest-cases~=3.6
+    pytest-cov~=2.12
+    pytest_randomly~=3.8
+    types-python-dateutil
+    types-pytz
+    wheel~=0.36
+doc =
+    autoapi
+    sphinx-autodoc-typehints
+    sphinx_rtd_theme
```

Comment:
Could you please add version numbers (either with >= or ~=) to the few python packages where none are defined